build.gradle

```diff
@@ -42,12 +42,13 @@ task compileModules(type: JavaCompile) {
     dependsOn compileTestJava
     source = fileTree(dir: 'src/classes/modules')
     classpath = files('build/annotations','build/classes','build/main')
-    options.compilerArgs = [  
-        '--module-source-path', file('src/classes/modules'),
```

Comment:
Please try to revert all the changes that only modify whitespace, as it makes it harder to analyze the patch in the future (or to merge it with other branches). The change should be one line (the last line).